Makefile

```diff
@@ -301,16 +301,16 @@ $(KAVM_LIB)/$(avm_kompiled): plugin-deps $(hook_includes) $(avm_includes) $(KAVM
 	@rm -f $(KAVM_DEFINITION_DIR)/interpreter.o # make sure the llvm interpreter gets rebuilt
 	@rm -f $(KAVM_DEFINITION_DIR)/interpreter
 	$(VENV_ACTIVATE) && $(KAVM) kompile $(KAVM_INCLUDE)/kframework/$(avm_main_file) \
-		                        --backend llvm                                              \
+                            --backend llvm                                              \
                             -I "${KAVM_INCLUDE}/kframework"                             \
                             -I "${plugin_include}/kframework"                           \
                             --definition-dir "${KAVM_LIB}/${avm_kompiled}"              \
                             --main-module $(avm_main_module)                            \
                             --syntax-module $(avm_syntax_module)                        \
                             --hook-namespaces KRYPTO KAVM                               \
                             --hook-cpp-files $(HOOK_KAVM_FILES) $(PLUGIN_CPP_FILES)     \
-                            --hook-clang-flags $(HOOK_CC_OPTS)
-
+                            --hook-clang-flags $(HOOK_CC_OPTS)                          \
+                            --coverage
```

Comment:
@geo2a FYI long-term you don't want the production build to have `--coverage`, so better to build the report with a makefile flag like `COVERAGE=true` or something. 

It does slow down the semantics.

Right now, I'm guessing execution time is not the bottleneck, so no need to worry. But something to keep in mind.